service/share/backoff.go

```diff
@@ -11,27 +11,33 @@ import (
 	"github.com/libp2p/go-libp2p/p2p/discovery/backoff"
 )
 
+// defaultTimeToLive is a default period after which disconnected peers will be removed from cache
+const defaultTimeToLive = time.Hour
```

Comment:
can just do `defaultTTL`

Reply:
sure

Reply:
Well, calling this TTL or TimeToLive is not correct. TTLs are attached to a specific entity/object with some living time, while here, we have a periodic duration for GC.